apps/web/composables/useNewsletter/useNewsletter.ts

```diff
@@ -39,8 +44,18 @@ export const useNewsletter: UseNewsletterReturn = () => {
     return !!data.value?.data;
   };
 
+  const confirmEmail: EmailConfirmationOptin = async (newsletterEmailId, authString) => {
+    const { data, error } = await useAsyncData(() =>
+      useSdk().plentysystems.doEmailConfirmation({
+        newsletterEmailId: newsletterEmailId,
+        authString: authString,
+      }),
+    );
```

Comment:
when ever we use a restcall that is only executed on client side we don't want to use the useAsyncData. This should only be used on server-side calls. useSdk().... can be used without the useAsyncData wrapper.

```suggestion
     return useSdk().plentysystems.doEmailConfirmation({
        newsletterEmailId: newsletterEmailId,
        authString: authString,
      });
```